Art/daniel1david/art_style.css

```diff
@@ -0,0 +1,195 @@
+body {
+background-image: url('https://cdn1.vectorstock.com/i/1000x1000/86/10/background-with-jungle-plants-vector-23578610.jpg');
+background-repeat: no-repeat;
+background-size: cover;
+}
+
+.box {
+	position: relative;
+	margin: auto;
+	display: block;
+	margin-top: 8%;
+	width: 600px;
+	height: 420px;
+/*	background: white;*/
```

Comment:
# Context
```js
/*	background: white;*/
```

# Feedback - unnecessary comment
Please remove the un used comment